### PR TITLE
Hao/landing nav bar

### PR DESCRIPTION
**Pull request by haophn01**
This is the navigation bar for Landing Page. 
Below is the description for this pull request:
- This brach is made base on the branch Hao/Page-Footer, so it should be accepted after you accept the Hao/Page-Footer first (jsut to avoid the merge conflict). 
- The new file called "LandingNavBar.tsx"  is created in the "ui" folder.
- All the inline styling and layout of how the nav bar should be look is all in 1 file.
- The file "Page.tsx" in "src" folder was modified.

### CODE REVIEW

File: client/src/ui/LandingNavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+/*
+ * This Landing Navbar will be responsible for rendering the Navbar of the Landing Page
+ */
+export function LandingNavbar() {
+  return (
+    <nav
+      style={{
+        background: "rgb(255, 255, 255)",
```
Comment (MekhiHart) [inline client/src/ui/LandingNavBar.tsx:16]: nit: Just do "white" to make it more obvious what the color is"

File: client/src/ui/LandingNavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        height: "80px",
+        width: "100%",
+        display: "grid",
+        zIndex: 999,
```
Comment (MekhiHart) [inline client/src/ui/LandingNavBar.tsx:20]: nit: Can you not just do 1 for this value to render on top?

File: client/src/ui/LandingNavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+              textAlign: "center",
+              justifyContent: "center",
+              alignItems: "center",
+              position: "absolute",
+              right: "5%",
+            }}
+          >
+            <li>
+              <Link
+                href="/aboutus"
```
Comment (MekhiHart) [inline client/src/ui/LandingNavBar.tsx:97]: route for this should be /about

**Review by MekhiHart — CHANGES_REQUESTED** [review]
I noticed that the branch has a lot of the same changes as the PR from the footer.  Make a new PR that implements the LandingNavBar component to not worry about merge conflicts

File: client/src/ui/LandingNavBar.tsx (hunk 2 above)
Comment (haophn01) [reply client/src/ui/LandingNavBar.tsx]: It has been fixed!

File: client/src/ui/LandingNavBar.tsx (hunk 1 above)
Comment (haophn01) [reply client/src/ui/LandingNavBar.tsx]: It has been fixed!

File: client/src/ui/LandingNavBar.tsx (hunk 3 above)
Comment (haophn01) [reply client/src/ui/LandingNavBar.tsx]: It has been fixed!

**Review by MekhiHart — CHANGES_REQUESTED** [review]

File: client/src/ui/LandingNavBar.tsx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+              <ThemeProvider theme={customTheme}>
+                <Button
+                  variant="contained"
+                  href="/signup"
+                  sx={{
+                    backgroundColor: "#496FFF",
+                    borderRadius: "8px",
+                    padding: "10px 60px",
+                    fontSize: "18px",
+                    "&:hover": {
```
[the remark this answers is not in the document]
Comment (haophn01) [reply client/src/ui/LandingNavBar.tsx]: It has been fixed!

File: client/src/ui/LandingNavBar.tsx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+                  color: "#3C435C",
+                  textDecoration: "none",
+                }}
+              >
+                <Typography>Log In</Typography>
+              </Link>
+            </li>
+
+            <li>
+              <ThemeProvider theme={customTheme}>
```
[the remark this answers is not in the document]
Comment (haophn01) [reply client/src/ui/LandingNavBar.tsx]: It has been removed!

File: client/src/ui/LandingNavBar.tsx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+    <nav
+      style={{
+        background: "white",
+        height: "80px",
+        width: "100%",
+        display: "grid",
+        zIndex: 1,
+        position: "sticky",
+        top: 0,
+        borderBottom: "5px solid #E0E4F2",
```
[the remark this answers is not in the document]
Comment (Serranoda) [reply client/src/ui/LandingNavBar.tsx:23]: noted

File: client/src/ui/LandingNavBar.tsx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+          alignItems: "center",
+        }}
+      >
+        <Link
+          href="/"
+          style={{
+            fontSize: "3rem",
+            fontFamily: "'Droid Sans', sans-serif",
+            color: "#3C435C",
+            position: "absolute",
```
[the remark this answers is not in the document]
Comment (Serranoda) [reply client/src/ui/LandingNavBar.tsx]: I intended to switch it to relative. The are fixed now

File: client/src/ui/LandingNavBar.tsx [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+                    borderRadius: "8px",
+                    padding: "10px 60px",
+                    fontSize: "18px",
+                    "&:hover": {
+                      backgroundColor: "#3C435C",
+                      transform: "scale(1.05)",
+                    },
+                  }}
+                >
+                  <Typography sx={{ color: "#FFFFFF" }}>Get Started</Typography>
```
[the remark this answers is not in the document]
Comment (Serranoda) [reply client/src/ui/LandingNavBar.tsx]: fixed

**Review by MekhiHart — APPROVED** [review]
Good job

**Pull request merged**